.github/workflows/pluto_build.yml

```diff
@@ -107,3 +116,29 @@ jobs:
       - name: Export and Upload
         run: ./06_export.sh
         if: inputs.run_export
+
```

Comment:
I'll say, this does feel a little misplaced. It would be nice if this were more generic and reusable, but since this is a single use-case for now, I left it here.